src/dmd/dsymbolsem.d

```diff
@@ -569,6 +569,29 @@ const(char)* getMessage(DeprecatedDeclaration dd)
     return dd.msgstr;
 }
 
+// Returns true if a contact can appear with a function body.
+private bool allowsContractWithoutBody(FuncDeclaration funcdecl)
```

Comment:
the name of the function appears to contradict the `Returns ...` in the function comment.